src/threat_model/secure_deployment.md

```diff
@@ -0,0 +1,32 @@
+# Recommendations on a Secure Parsec Deployment
+
+The following recommendations should be applied in order for Parsec to respect some of the
+operational mitigations. These recommendations have been thought of for a Linux system and should be
+modified/ported for other operating systems. The linked [operational
+mitigations](threat_model.md#operational-mitigations) are noted with `O-n`.
+
+- The service needs to be running as a separate, trusted, user. Ideally the service would be running
+   as the `parsec` user (O-0).
```

Comment:
I think if we do go and implement the check described in the last point, we have to be more strict here and say "The service *must* be running as the `parsec` user."

We should also add some notice of this somewhere in the Linux installation guide, if client libraries won't connect at all if these steps arent' taken

Reply:
Will change! Will also add a notice on the clients saying that those steps are essential for clients to successfully connect and verify the service.